server/api/controllers/user.js

```diff
@@ -42,18 +41,19 @@ export const registerUser = async (req, res) => {
       email,
       password,
     });
+    const link = `${req.protocol}://${req.headers.host}/api/users/${createdUser.id}`;
 
     try {
-      await sendVerificationMail(username, email, createdUser.id);
+      await sendVerificationMail(username, email, link);
     } catch (error) {
       logger.debug('Email Error::', error);
     }
 
     return res.status(201).send({
       status: 'success',
       data: {
-        message: `A confirmation email has been sent to ${email}. Click on the confirmation button to verify the account`,
-        link: `${req.protocol}://${req.headers.host}/api/users/${createdUser.id}`
+        user: `A confirmation email has been sent to ${email}. Click on the confirmation button to verify the account`,
```

Comment:
@Ibidapo Please change the `user` key in the response payload on line `55` to `message`.